otx/algorithms/detection/adapters/mmdet/models/backbones/mmov_backbone.py

```diff
@@ -1,24 +1,29 @@
-# Copyright (C) 2022 Intel Corporation
+"""Backbone Class of OMZ model for mmdetection backbones."""
+# Copyright (C) 2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 #
 
 from mmdet.models.builder import BACKBONES
 
-from ...mmov_model import MMOVModel
+from otx.mpa.modules.ov.models.mmov_model import MMOVModel
 
 
 @BACKBONES.register_module()
 class MMOVBackbone(MMOVModel):
+    """MMOVBackbone Class."""
+
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     def forward(self, *args, **kwargs):
+        """Forward function of MMOVBackbone."""
         outputs = super().forward(*args, **kwargs)
         if not isinstance(outputs, tuple):
             outputs = (outputs,)
         # must return tuple
         return outputs
 
     def init_weights(self, pretrained=None):
+        """Initial weights function of MMOVBackbone."""
         # TODO
-        pass
+        pass  # pylint: disable=unnecessary-pass
```

Comment:
Can we change this to return? I mean pass -> return

Reply:
I'm not sure if there's some other intent behind leaving it as pass, maybe I should fix that?

Reply:
@cih9088 Would it be okay to modify this to return?

Reply:
No problem.

Reply:
Fixed.